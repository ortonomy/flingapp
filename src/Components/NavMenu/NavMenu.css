.NavMenu {
  background: #333;
  height: 4em;
  display: flex;
  flex-flow: row nowrap;
  justify-content: flex-start;
  align-items: center;
}

.NavMenu .nav-menu-item {
  height: 100%;
  width: 10em;
  text-transform: uppercase;
  color: white;
  display: flex;
  justify-content: center;
  align-items: center;
}

.NavMenu .nav-menu-item.active {
  background: #FFDE17;
  color: black;
}
