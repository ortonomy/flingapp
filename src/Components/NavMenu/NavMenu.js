import React, { Component } from 'react';
import './NavMenu.css';

class NavMenu extends Component {
  render() {
    return (
      <div className="NavMenu">
        <div className="active nav-menu-item">Freelancers</div>
        <div className="nav-menu-item">Projects</div>
      </div>
    )
  }
}

export default NavMenu;
