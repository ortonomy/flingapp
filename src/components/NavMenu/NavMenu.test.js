import React from 'react';
import NavMenu from '../NavMenu';

it('renders without crashing', () => {
  const wrapper = shallow(<NavMenu />);
  expect(wrapper).toMatchSnapshot();
});
