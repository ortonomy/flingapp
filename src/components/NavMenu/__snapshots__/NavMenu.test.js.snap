// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NavMenu />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className={undefined}
>
          Freelancers
</div>,
        <div
          className={undefined}
>
          Projects
</div>,
      ],
      "className": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Freelancers",
          "className": undefined,
        },
        "ref": null,
        "rendered": "Freelancers",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Projects",
          "className": undefined,
        },
        "ref": null,
        "rendered": "Projects",
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className={undefined}
>
            Freelancers
</div>,
          <div
            className={undefined}
>
            Projects
</div>,
        ],
        "className": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Freelancers",
            "className": undefined,
          },
          "ref": null,
          "rendered": "Freelancers",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Projects",
            "className": undefined,
          },
          "ref": null,
          "rendered": "Projects",
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
